connectors/src/connectors/microsoft/temporal/file.ts

```diff
@@ -163,50 +151,26 @@ export async function syncOneFile({
     );
   }
 
-  async function getDocumentContent() {
-    try {
-      const converted = await mammoth.convertToHtml({
-        buffer: Buffer.from(downloadRes.data),
-      });
-
-      const extracted = new turndown()
-        .remove(["style", "script", "iframe", "noscript", "form", "img"])
-        .turndown(converted.value);
-
-      return extracted.trim();
-    } catch (err) {
-      localLogger.error(
-        {
-          error: err,
-        },
-        `Error while converting docx document to text`
-      );
-      throw err;
-    }
-  }
-
   let documentSection: CoreAPIDataSourceDocumentSection | null = null;
   if (
-    mimeType ===
-    "application/vnd.openxmlformats-officedocument.presentationml.presentation"
+    [
```

Comment:
nit: Use the same flow as google - handle all specific cases first  - csv, excel, plaintext, then handleTextExtraction in else case